front_cam_sn: '24799899'
back_cam_sn: '24360073'